### DISCUSSION TITLE
Проблема  при использовании с  firefox spoof-dpi  на  openwrt mips.

### DISCUSSION THREAD

**Discussion by vehametra** [xvzc/SpoofDPI#274] [Q&A]
![Снимок экрана_2024-10-13_13-06-03](https://github.com/user-attachments/assets/0f9c8f6f-babd-41e0-9de0-68800b8614ca)

В chrome работает, на андройд через ### "Super Proxy"  тоже.

**Comment by Gen5551** [comment]
Хрена себе, вы в роутер запихали прокси? Молодцы!
